portal-api/src/v1/gef/controllers/application-submit.js

```diff
@@ -107,15 +100,17 @@ const updateChangedToIssued = async (dealId) => {
 const addUkefFacilityIdToFacilities = async (dealId) => {
   const facilities = await getAllFacilitiesByDealId(dealId);
 
-  await Promise.all(facilities.map(async (facility) => {
-    if (!facility.ukefFacilityId) {
-      const { ukefId } = await generateUkefFacilityId(facility);
-      const update = {
-        ukefFacilityId: ukefId,
-      };
-      await updateFacility(facility._id, update);
-    }
-  }));
+  await Promise.all(
+    facilities.map(async (facility) => {
+      if (!facility.ukefFacilityId) {
```

Comment:
This isn't for a code comment but do you think it's worth having some documentatino somewhere on the submission process? This PR makes everything so much better and neater but it's still a bit confusing on when and where everything should get IDs then get submitted to TFM and have the portal side snapshotted in TFM etc.
Might help for new joiners to know the process of going portal -> TFM

Reply:
Good shout, will add.